Package/Sublime Text Color Scheme/Sublime Text Color Scheme.sublime-settings

```diff
@@ -4,4 +4,6 @@
     [
         {"characters": ".", "selector": "meta.scope-selector.sublime"}
     ],
+    "trim_trailing_white_space_on_save": true,
```

Comment:
Why add this setting? It's not mandated by the syntax/format and doesn't have any benefit (other than generally being the correct default, but that's not for me to decide).

Same for the theme.

Reply:
https://github.com/SublimeText/PackageDev/blob/7f470db736a40e4789f954dd7ee555b34d91d4cb/Package/Sublime%20Text%20Syntax%20Definition/Sublime%20Text%20Syntax%20Definition.sublime-settings#L14-L15

Reply:
Syntax definitions are written in YAML, not JSON, where trailing whitespace is ignored (except inside block scalars) and that's not immediately obvious to users. Trimming trailing whitespace *can* help prevent anti-patterns here, I guess.

Though you do have a point in that this *probably* doesn't really bring much to the table either. I'd rather remove whitespace trimming there as well than introducing it to the other syntaxes, however.